scripts/prepdocs.py

```diff
@@ -234,6 +244,12 @@ def compute_embedding(text):
     refresh_openai_token()
     return openai.Embedding.create(engine=args.openaideployment, input=text)["data"][0]["embedding"]
 
+@retry(wait=wait_random_exponential(min=1, max=60), stop=stop_after_attempt(15), before_sleep=before_retry_sleep)
```

Comment:
Suggestion from @charris-msft : Increase this min to 15? His deploy eventually errored out due to too many attempts (> 15). We're thinking that if it always waits 15 seconds, it's less likely to eventually error out.